bears/general/TextLintBear.py

```diff
@@ -0,0 +1,163 @@
+                    NpmRequirement('textlint-rule-ng-word', '1.0.0'),
+                    NpmRequirement('textlint-rule-no-dead-link', '3.1.1'),
+                    NpmRequirement('textlint-rule-no-empty-section', '1.1.0'),
+                    NpmRequirement('textlint-rule-no-start-'
+                                   'duplicated-conjunction', '1.1.3'),
+                    NpmRequirement('textlint-rule-no-todo', '2.0.0'),
+                    NpmRequirement('textlint-rule-period-in-list-item',
+                                   '0.2.0'),
+                    NpmRequirement('textlint-rule-rousseau', '1.4.5'),
+                    NpmRequirement('textlint-rule-unexpanded-acronym',
+                                   '1.2.1'),
+                    NpmRequirement('textlint-rule-write-good', '1.6.0'),
+                    PipRequirement('docutils-ast-writer', '0.1.2')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting', 'Grammar', 'Spelling'}
+    SEE_MORE = 'https://github.com/textlint/textlint'
+
+    @staticmethod
+    def generate_config(filename, file,
+                        keyword_todo: bool=None,
+                        no_start_duplicated_conjunction: bool=True,
+                        no_empty_section: bool=True,
+                        check_date_weekday_mismatch: bool=True,
+                        ginger: bool=True,
+                        max_lines_per_file: int=300,
+                        max_comma_per_sentence: int=4,
+                        check_ng_words: typed_list(str)=[],
+                        period_in_list_item: bool=True,
+                        minimum_acronym_length: int=3,
+                        maximum_acronym_length: int=5,
+                        ignore_acronyms: typed_list(str)=[],
+                        check_rousseau: bool=True,
+                        check_alex: bool=True,
+                        check_common_misspellings: bool=True,
+                        write_good: bool=True,
+                        check_invalid_link: bool=True,
+                        textlint_config: str='',
+                        ):
+        """
+        :param keyword_todo:
+            This rule checks ``- [ ]`` TODO text.
+        :param no_start_duplicated_conjunction:
+            This rule checks whether your sentence starts with a duplicated
+            conjunction.
+        :param no_empty_section:
+            This rule does not allow to create an empty section.
+            For example, there is an empty section ``# Header B`` below:
+            ```
+            # Header A
+
+            text.
+
+            # Header B
+            ```
+        :param check_date_weekday_mismatch:
+            This rule finds a mismatch between date and the corresponding
+            weekday.
+        :param ginger:
+            This rule checks your English grammar with Ginger Proofreading.
```

Comment:
rename the setting, don't call it `ginger` just because the plugin is called like that ;D Setting names shall resemble what they intend to fix/check :)